frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -178,7 +196,19 @@ pub mod pallet {
         T::SmartContract,
         Twox64Concat,
         EraIndex,
-        EraStakingPoints<T::AccountId, BalanceOf<T>>,
+        ContractStakeInfo<BalanceOf<T>>,
+    >;
+
+    #[pallet::storage]
+    #[pallet::getter(fn staker_info)]
+    pub(crate) type StakersInfo<T: Config> = StorageDoubleMap<
```

Comment:
May be rename `StakersInfo` -> `GeneralStakerInfo`, because of possible types `Stakers` = `Staker`

Reply:
Sure, I can do that.

@impelcrypto please not that I will change this in the new version.